packages/@react-facet/core/src/index.ts

```diff
@@ -7,3 +7,4 @@ export * from './helpers'
 export * from './hooks'
 export * from './mapFacets'
 export * from './types'
+export { batch } from './scheduler'
```

Comment:
Let's remain consistent and export with `*`. I know there are other two functions being exported and maybe they were meant as internal, but I think there's value in exposing them, we don't yet know if there is no use for them in third party library code.